services/alarm-server/src/main/java/org/phoebus/applications/alarm/server/AlarmServerMain.java

```diff
@@ -137,8 +137,8 @@ private boolean handleShellCommands(final String... args) throws Throwable
             restart.offer(false);
         else if (args.length == 1)
         {
-            if (args[0].equals("shutdown"))
-                restart.offer(false);
+	    if (args[0].startsWith("shut"))
```

Comment:
For what it's worth, try to use N*4 spaces for indentation, not tabs, nor a mixture of tabs/spaces.
But that's minor, and there's no agreement on style anyway, so go ahead.

Reply:
I am very bad about styles, can you explain "N*4 spaces" ?

Reply:
That file intents by 0, 4, 8, .., multiples of 4 spaces.
The `if (args[0].startsWith("shut"))` should be intended twice (8 spaces) to align with the rest. Looks like it's using "tab" plus 3 spaces. The width of a tab could be 4 or 8, who knows.

Reply:
I do my best to match what you described, please let me know what you think.

Reply:
@kasemir , I have few more questions on styles.

-  How do you make *N*4 spaces"? Do you type them all with space key?
-  Do you prefer this style in Java?  I would like not to break your style while I change / update codes.

Reply:
I set Eclipse to use spaces for indentation, specifically 4.
Most IDEs have settings for that.
In `vi` I do type 4 spaces.

This is getting ridiculous, just merge it, I'm the only one who cares about this anyway.

Reply:
@kasemir , Sorry for these commits, but I will do this later according to what you describe as much as I can. I can see clearly why you try to do this while looking at github filechanges, which gives me more clean view.  I like it.